htdocs/web_portal/components/Draw_Components/draw_page_components.php

```diff
@@ -118,10 +114,17 @@ function Get_User_Info_Box_HTML()
     }
 
     /* Draws a box showing the EGI and other logos */
-    function Get_bottom_logos_Box_HTML()
+    function Get_top_logos_Box_HTML()
     {
         $HTML = "";
         $HTML .= '<div class="Left_Logo_Box left_box_menu">';
+        $HTML .= Get_File_Contents(__DIR__."/../../static_html/goc5_logo.html");
+        //Insert a portal is in read only warning message, if it is
```

Comment:
Could the "portal is in read only" bit be quoted as it's hard to parse this comment otherwise.

Reply:
Fixed it myself.